source/Halibut.TestUtils.CompatBinary.Base/SettingsHelper.cs

```diff
@@ -39,7 +39,7 @@ int AsInt(string s)
 
         public static string GetSetting(string name)
         {
-            return Environment.GetEnvironmentVariable(name);
+            return Environment.GetEnvironmentVariable(name) ?? string.Empty;
```

Comment:
nit: null might be more meaningful when the setting is not set rather than empty.

Reply:
The motivation was more that everywhere that uses `GetSetting` expects there to be a value. I want to keep that assumption true.

An alternative, could be to throw a `NullReferenceException` (or something else) instead? That way failures are loud and proud?

Reply:
leave as is